api/templates/spotify.compact.html.j2

```diff
@@ -1,4 +1,5 @@
-<svg width="320" height="{{height}}" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="320" height="{{height}}" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-labelledby="catTitle">
+  <title id="catTitle">Now playing on Spotify</title>
```

Comment:
Why is it `catTitle`? This template renders a card, should it be `cardTitle` or something else instead?

Reply:
Great catch! Yes, `cardTitle` would be much better and was the intended name. I'll get that change.